.circleci/config.yml

```diff
@@ -99,6 +99,10 @@ commands:
           command: |
             ./embuilder build MINIMAL --lto
             ./tests/runner test_hello_world
+      - run:
+          name: embuilder (WASM64)
+          command: |
+            ./embuilder build MINIMAL --wasm64
```

Comment:
Could we maybe build this only on say linux? Just to avoid extra CI work.

Reply:
build-libs-and-freeze already only runs on linux I think.   On other platform we just build libs on demand.

Reply:
Oh, sounds good then.